Library/Formula/graphviz.rb

```diff
@@ -43,7 +43,7 @@ class Graphviz < Formula
 
   if build.with? "bindings"
     depends_on "swig" => :build
-    depends_on :python
+    depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Ugh sorry. I missed that. It seems to have built the bindings and put them in the cellar, but didn't install them. What's the workaround?